cmd/core-data/res/configuration.yaml

```diff
@@ -9,6 +9,7 @@ Writable:
 #    Tags: # Contains the service level tags to be attached to all the service's metrics
     ##    Gateway="my-iot-gateway" # Tag must be added here or via Consul Env Override can only change existing value, not added new ones.
   EventPurge: false # Remove the related events and readings once received the device deletion system event
+  DefaultQueryOffset: 0
```

Comment:
Why do we allow users to configure their preferable default query offset, but not other query parameters?

Moreover, the go-mod-core-contracts actually defines DefaultOffset. https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/common/constants.go#L216

Reply:
If we allow users to configure the default query offset, the user can query the API without putting the parameter.